ad9361/sw/ad9361.c

```diff
@@ -4346,12 +4350,12 @@ static int32_t ad9361_validate_trx_clock_chain(struct ad9361_rf_phy *phy,
 	}
 
 	for (i = 1; i <= 3; i++) {
-		if (abs(rx_path_clks[ADC_FREQ] / i - data_clk) < 4)
+		if (labs((int64_t)(rx_path_clks[ADC_FREQ] / (i - (int64_t)data_clk))) < 4)
```

Comment:
i am starting to think here that maybe it's simpler/easier to just convert both `i` & `data_clk` to int64_t;
it would simplify the nature of these operations and remove some of the explicit casts;